app/contrib/campaign/cms_wizards.py

```diff
@@ -48,13 +48,15 @@
 #         return has_perm
 
 
-# cms_pressure_wizard = CMSPressureWizard(
-#     title="Campanha de Pressão",
-#     weight=100,
-#     form=CreatePressureForm,
-#     model=Page,
-#     description="Pressão por e-mail Explica tática"
-# )
+cms_pressure_wizard = CMSPressureWizard(
+    title="Campanha de Pressão",
+    weight=100,
+    form=CreatePressureForm,
+    model=Page,
+    description="Pressão por e-mail Explica tática"
```

Comment:
```suggestion
    description="Pressão por e-mail Explica tática"
```
uma duvida: o que Explica tática significa? 🤔

Reply:
@miguelzinh3, acho que era pra explicar a Tática da Pressão por e-mail nesse lugar